presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,7 +14,90 @@ make it executable with ``chmod +x``, then run it:
 
     ./presto --server localhost:8080 --catalog hive --schema default
 
-Run the CLI with the ``--help`` option to see the available options.
+The available options for the CLI are:
```

Comment:
We should move the options to a separate section at the end, so as not to obscure the paragraph about the pager.

I'm also thinking these should be organized according to usage. For example, catalog, schema, source, etc. should go together. So should the kerberos options.